src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts

```diff
@@ -132,3 +135,44 @@ suite("Backend", function () {
     });
   });
 });
+
+suite("EdgeTPUCompiler", function () {
+  suite("#parseVersion", function () {
+    test("returns Version object from string version", function () {
+      const edgeTPUCompiler = new EdgeTPUCompiler();
+      const version1 = edgeTPUCompiler.parseVersion("1.0.2~RC0");
+      const version2 = new Version(1, 0, 2, "~RC0");
+      assert.deepEqual(version1, version2);
+    });
+    test("returns Version object from string version without patch and option", function () {
+      const edgeTPUCompiler = new EdgeTPUCompiler();
+      const version1 = edgeTPUCompiler.parseVersion("16.0");
+      const version2 = new Version(16, 0);
+      assert.deepEqual(version1, version2);
+    });
+    test("returns Version object from string version without option", function () {
+      const edgeTPUCompiler = new EdgeTPUCompiler();
+      const version1 = edgeTPUCompiler.parseVersion("2.1.302470888");
+      const version2 = new Version(2, 1, 302470888);
+      assert.deepEqual(version1, version2);
+    });
+    test("returns Version object from string version without minor", function () {
```

Comment:
Umm... I think we should change the test title to "returns Version object from string version without patch".